### DISCUSSION TITLE
3d object detection from pointcloud 3d -is there a demo?

### DISCUSSION THREAD

**Discussion by AndreV84** [scalabel/scalabel#491] [General]
Hi, my concern is to find out if there is a vizualizeable demo that allows to use scalabel not to just manually annotate 3d pointcloud, but just run an existing model on 3d pointcloud input in a way so that it will detect 3d objects that will be visualized somehow.
Thanks

**Comment by tobiasfshr** [comment]
There isn't a demo so far, but what you can do is to add the 3D bounding boxes of your predicted objects to the labels of the respective frames in the json file that you use to create your project. The [format documentation](https://doc.scalabel.ai/format.html) shows how it is defined. This way your predictions will be visualized as existing labels in the web interface.
